defaults/main.yml

```diff
@@ -113,4 +117,5 @@ sympa_default_home: home
 sympa_edit_list: owner
 sympa_ldap_force_canonical_email: 1
 sympa_review_page_size: 25
+sympa_webserver_type: Other                        # 'Other' or 'Apache 2'
```

Comment:
lets use lowercase strings without a space

Reply:
Well, this variables is there to set a debconf parameter of `select` type. When you install sympa manually, and after checking the debconf db with `debconf-get-selections` it clearly proposes "Other" and "Apache 2". 

I did some test, pre-setting the parameter with: 
```
echo "sympa wwsympa/webserver_type select <value>" | debconf-set-selections
``` 
trying with `<value>` set to `Other`, `other`, `apache2`, `Apache 2`, etc. 

Without changing debconf priorities, when setting the value to `Other` it silently preserves this value. When setting other values, it silently overwrites to the default value `Apache 2`. 

So I think that there, we have to stick to the exact spelling proposed for the two debconf `select` options. 

BTW, I [struggled a lot with sympa debconf parameter](https://github.com/UdelaRInterior/ansible-sympa/issues/3#issuecomment-590107152). Sympa has a quite peculiar behaviour: most of the parameters are low o medium priority (so they aren't asked for), several are "internal", and most of all, they seem to be heavily overwritten at installation with default values, eventually built from linux hostname.